### PR TITLE
Add bitbrain tutorials :memo:

### PR DESCRIPTION
**Pull request by bitbrain**
thumbnail

Hello everyone!

My name is Miguel (aka [bitbrain](https://www.youtube.com/channel/UCZDjQltHRNiXIYXMBeLDleA)) and I recently started doing devlogs and tutorials! I added my name to the list as I aim to produce monthly new content (devlogs + tutorials) and this might be interesting to some folks for sure.

### CODE REVIEW

**Comment by bitbrain** [comment]
The following timed out:
```
⌛ https://itch.io/app
⌛ https://ifiregames.itch.io/stern-flowers-chibifire-com-godot-engine
```
could we re-run the pipeline?

File: README.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 - [HeartBeast](https://www.youtube.com/user/uheartbeast) - A Godot developer's diary and tutorials.
 - [KidsCanCode](https://www.youtube.com/channel/UCNaPQ5uLX5iIEHUCLmfAgKg) - Tutorials about 2D and 3D game development.
 - [Miziziziz's channel](https://www.youtube.com/user/Miziziziz) - Tutorials about 3D effects and replicating concepts from well-known games.
+- [bitbrain](https://youtube.com/bitbraindev) - Devlogs and tutorials about building various concepts with Godot.
```
Comment (Calinou) [inline README.md]: This line should be moved between Bastiaan Olij's channel and Bitwes's channel to respect alphabetical order.

**Review by Calinou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bitbrain) [reply README.md]: fixed.

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Pull request merged**